homeassistant/components/climate/demo.py

```diff
@@ -47,6 +51,15 @@ def __init__(self, name, target_temperature, unit_of_measurement,
         self._target_temperature_high = target_temp_high
         self._target_temperature_low = target_temp_low
 
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return (SUPPORT_TARGET_TEMPERATURE | SUPPORT_TARGET_HUMIDITY |
```

Comment:
Where possible, move this math to constants so we aren't calculating it every time we check.

https://github.com/home-assistant/home-assistant/blob/d8bf15a2f5f5d15aaca0223e8afd384d6771a5aa/homeassistant/components/media_player/cast.py#L32-L34

Reply:
Ah yes, the C++ guy in me assumed these to be computed at compile time. Will change that!